### ISSUE TITLE
VMSS image upgrades on Bronze node types

### ISSUE THREAD

**Issue by hbuckle** [Azure/service-fabric-issues#1634]
We are running our stateless workloads on node types with Bronze durability level so we can quickly add and remove instances.
We would like to deploy a new image version to the VMSS and want to use the Rolling upgrade policy so that all nodes are not upgraded at the same time. This requires a health probe for the VMSS, but trying to add one produces `Error: Health probe is not supported on Service Fabric Virtual Machine Scale Set.`

So how are we supposed to upgrade the VMSS image on Bronze node types without them all happening at the same time or doing it manually?

**Comment by anantshankar17** [comment]
For now you can use POA (https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-patch-orchestration-application) to orchestrate VM OS upgrades on nodetypes with bronze durability.
We are working on supporting health probes on bronze SF nodetypes, which would run only stateless workloads.

**Comment by hbuckle** [comment]
It looks like POA just patches the existing OS, rather than upgrading the VMSS image though?
I also thought tried setting the upgrade policy to manual then we could control the upgrade ourselves, but that is also not allowed

```
Code="OperationNotAllowed" Virtual Machine Scale Sets with extension Microsoft.Azure.ServiceFabric.ServiceFabricNode must be set to automatic upgrade mode.
```

**Comment by raunakpandya** [comment]
Yes, currently Automatic VMSS OS Upgrade is not allowed for Bronze durability tiers. To keep your OS patched, you would need to rely on POA.

**Comment by hbuckle** [comment]
Is there any timeline for supporting health probes or manual upgrades?
Currently Bronze node types are pretty pointless because Service Fabric blocks you from using any of the features that would make them useful.
